<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window</class>
 <widget class="QTabWidget" name="window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Supybot GUI</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="configurationTab">
   <attribute name="title">
    <string>Configuration</string>
   </attribute>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="configurationTreeLayout">
      <item>
       <widget class="QTreeView" name="configurationTree"/>
      </item>
      <item>
       <widget class="QPushButton" name="refreshConfigurationTree">
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="configurationDetailsLayout">
      <item>
       <layout class="QVBoxLayout" name="configurationEditLayout">
        <item>
         <widget class="QLabel" name="configurationVariableLabel">
          <property name="text">
           <string>Value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="configurationValueEdit"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="configurationEditButtonsLayout">
          <item>
           <widget class="QPushButton" name="configurationDefaultButton">
            <property name="text">
             <string>Default</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="configurationSetButton">
            <property name="text">
             <string>Set</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="configurationHelpLabel">
        <property name="text">
         <string>Help</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="configurationHelp"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="commandsTab">
   <attribute name="title">
    <string>Commands</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPlainTextEdit" name="commandsHistory">
      <property name="textInteractionFlags">
       <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="commandsWritingLayout">
      <item>
       <widget class="QLineEdit" name="commandEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="commandSend">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
